`make` fails on centos7

I've installed the podman rpm (`podman-0.9.2-5.git37a2afe.el7_5.x86_64`) to ensure all dependencies are present, and then built&installed podman master

```bash
$ podman version
Version:       0.10.2-dev
Go Version:    go1.11
Git Commit:    "76360d9a6ef73f694138766b81f380e45a860556"
Built:         Mon Nov  5 12:44:35 2018
OS/Arch:       linux/amd64
```

When I attempt to `make` skopeo master (187299a20b864610a55711278856b7a15ae4e37a) it fails at this point:
```
STEP 7: COMMIT skopeobuildimage
/usr/local/bin/podman run --rm --security-opt label:disable -v $(pwd):/src/github.com/containers/skopeo \
        skopeobuildimage make binary-local  BUILDTAGS='  '
Invalid --security-opt 1: "label:disable"
make: *** [binary] Error 125
```

--- a/Makefile
+++ b/Makefile
@@ -70,12 +70,12 @@ all: binary docs
 # Then do the build and the output (skopeo) should appear in current dir
 binary: cmd/skopeo
 	${CONTAINER_RUNTIME} build ${BUILD_ARGS} -f Dockerfile.build -t skopeobuildimage .
-	${CONTAINER_RUNTIME} run --rm --security-opt label:disable -v $$(pwd):/src/github.com/containers/skopeo \
+	${CONTAINER_RUNTIME} run --rm --security-opt label=disable -v $$(pwd):/src/github.com/containers/skopeo \
 		skopeobuildimage make binary-local $(if $(DEBUG),DEBUG=$(DEBUG)) BUILDTAGS='$(BUILDTAGS)'
 
 binary-static: cmd/skopeo
 	${CONTAINER_RUNTIME} build ${BUILD_ARGS} -f Dockerfile.build -t skopeobuildimage .
-	${CONTAINER_RUNTIME} run --rm --security-opt label:disable -v $$(pwd):/src/github.com/containers/skopeo \
+	${CONTAINER_RUNTIME} run --rm --security-opt label=disable -v $$(pwd):/src/github.com/containers/skopeo \
 		skopeobuildimage make binary-local-static $(if $(DEBUG),DEBUG=$(DEBUG)) BUILDTAGS='$(BUILDTAGS)'
 
 # Build w/o using containers

